src/olympia/accounts/tasks.py

```diff
@@ -48,6 +50,11 @@ def primary_email_change_event(profile, changed_date, email):
 @user_profile_from_uid
 def delete_user_event(user, deleted_date):
     """Process the delete user event."""
-    user.delete(addon_msg='Deleted via FxA account deletion')
-    log.info(
-        'Account pk [%s] deleted from FxA on %s' % (user.id, deleted_date))
+    if switch_is_active('fxa-account-delete'):
+        user.delete(addon_msg='Deleted via FxA account deletion')
+        log.info(
+            'Account pk [%s] deleted from FxA on %s' % (user.id, deleted_date))
+    else:
+        log.info(
+            f'Skipping deletion from FxA for account [{user.id}] because '
+            'waffle inactive')
```

Comment:
It matters less for things that don't go in sentry, so that doesn't block this PR, but we should try to always let `logging` to the formatting to allow for grouping - So we should do logging formatting with `log.info('foo %s', bar)` and not `log.info('foo %s' % bar)`, `log.info('foo {bar}')` or `log.info('foo {}'.format(bar))` where possible.